packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.tsx

```diff
@@ -40,7 +41,7 @@ export const TagItemBase = (props: ITagItemProps) => {
       data-selection-index={index}
       data-is-focusable={(enableTagFocusInDisabledPicker || !disabled) && true}
     >
-      <span className={classNames.text} aria-label={children as string} title={children as string}>
+      <span className={classNames.text} aria-label={title} title={title}>
```

Comment:
For consistency, children should still be passed into aria-label. That's where the string is expected to come in from and I think it helps keep that inline.

Reply:
aria-label has the same constraint than the title where only a string should be passed in. 
Should `children` be passed if they represent a string as a default value, falling back to `props.item.name`, while still being overridable by `props.title`?

Reply:
I have changed the default value logic to keep the previous behaviour. What do you think?

Reply:
Looks good!